media-source-respec.html

```diff
@@ -1397,9 +1455,25 @@ <h4><dfn>Initialization Segment Received</dfn></h4>
                 <li>Verify the following properties. If any of the checks fail then run the [=append error=] algorithm and abort these steps.
                   <ul>
                     <li>The number of audio, video, and text tracks match what was in the first [=initialization segment=].</li>
-                    <li>The codecs for each track, match what was specified in the first [=initialization segment=].</li>
-                    <li>If more than one track for a single type is present (e.g., 2 audio tracks), then the [=Track IDs=] match the ones in the
+                    <li>If more than one track for a single type are present (e.g., 2 audio tracks), then the [=Track IDs=] match the ones in the
                       first [=initialization segment=].</li>
+                    <li>The codecs for each track are supported by the user agent.
+                      <p class="note">User agents MAY consider codecs, that would otherwise be supported, as "not supported" here if the codecs were not
+                        specified in <var>type</var> parameter passed to
+                        (a) the most recently successful {{SourceBuffer/changeType()}} on this {{SourceBuffer}} object, or
+                        (b) if no successful {{SourceBuffer/changeType()}} has yet occurred on this object, the {{MediaSource/addSourceBuffer()}}
+                        that created this {{SourceBuffer}} object.
+                        For example, if the most recently successful {{SourceBuffer/changeType()}} was called with 'video/webm'
```

Comment:
I would wrap string parameters in `<code>`, as in:
```suggestion
                        For example, if the most recently successful {{SourceBuffer/changeType()}} was called with <code>'video/webm'</code>
```